crates/redstone/src/types/signer_address.rs

```diff
@@ -5,17 +5,17 @@ use crate::types::{Sanitized, VALUE_SIZE};
 /// Type describing address of signer. Typically pubkey of length 20 bytes;
 /// As of right now we dont expect larger keys than 32 bytes.
 /// The address is normalized to contain only lowercase letters (A-F) -> (a-f).
-#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+#[derive(Clone, Copy, PartialEq, Eq, Debug, Ord, PartialOrd)]
 #[cfg_attr(feature = "radix", derive(ScryptoSbor))]
-pub struct SignerAddress([u8; VALUE_SIZE]);
+pub struct SignerAddress(pub(crate) [u8; VALUE_SIZE]);
```

Comment:
Hmm, maybe implement something like asref<[u8]> instead of making this pub(crate) <- as this can violate the assumption that all `letters` are in lowercase.

Reply:
Thanks. Good point.